paimon-web-ui/src/views/metadata/components/table-column-content/index.tsx

```diff
@@ -252,9 +252,21 @@ export default defineComponent({
   },
   render() {
     return (
-      <vue-draggable v-model:value={this.data} animation={150} handle=".drag-handle" target=".n-data-table-tbody">
-        <n-data-table columns={this.columns} data={this.data} style={{ marginBottom: '24px' }} />
-      </vue-draggable>
+      h(
+        VueDraggable,
+        {
+          'modelValue': this.modelValue,
+          'onUpdate:modelValue': (value: any) => {
+            this.$props['onUpdate:modelValue']?.(value)
+          },
+          'animation': 150,
+          'handle': '.drag-handle',
+          'target': '.n-data-table-tbody',
+        },
+        {
+          default: () => <n-data-table columns={this.columns} data={this.modelValue} style={{ marginBottom: '24px' }} />,
```

Comment:
Why is the `marginBottom` 24px?

Reply:
Keep it some distance from the bottom of the popover box